src/libguac/guacamole/socket.h

```diff
@@ -136,11 +136,25 @@ void guac_socket_free(guac_socket* socket);
  * to ensure neither side of the socket times out while the socket is open.
  * This ping will take the form of a "nop" instruction.
  *
+ * @deprecated
+ *     Manually starting the keep alive process on sockets is no longer
+ *     necessary, as the sockets enable the "nop" ping by default.
+ * 
```

Comment:
This is no longer the case, correct? guacd will turn on the keep-alive ping automatically for the only socket that should generally require it (the broadcast socket), but this function no longer needs to be deprecated?